Scripts/GenerateSampleViewSourceCode.swift

```diff
@@ -41,6 +41,8 @@ private struct SampleMetadata: Decodable {
     let offlineData: [String]?
     /// The tags and relevant APIs of the sample.
     let keywords: [String]
+    /// The relevant APIs of the sample.
+    let relevantApis: [String]
```

Comment:
Based on this [doc](https://developer.apple.com/documentation/foundation/jsondecoder/keydecodingstrategy/convertfromsnakecase), `relevantAPIs` won't work, so this name is acceptable.